File: python/implementation_algorithms/time_count.py
#import time
#start_time = time.time()
# h입력 받기
h = int(input())
count = 0

# 3중 for문
for i in range(h+1):
    for j in range(60):
        for k in range(60):
            if '3' in str(i) + str(j) + str(k):
                count += 1

print(count)

#end_time = time.time()
#print("time : ", end_time - start_time)
#time = 0.7

File: Implementation-Algo-books/time_count.cpp
#include<iostream>
#include<algorithm>
#include<string>
#include<ctime>
using namespace std;

int main()
{
	time_t start_time = time(0);

	int N = 0;
	cin >> N;

	int count = 0;
	int hou = 0, min = 0, sec = 0;
	
	for (int hou = 0; hou < N+1; hou++)
	{
		for (int min = 0; min < 60; min++)
		{
			for (int sec = 0; sec < 60; sec++)
			{
				string houS = to_string(hou);
				string minS = to_string(min);
				string secS = to_string(sec);
				string bigTime = houS + minS + secS;

				if (bigTime.find("3") == string::npos)
				{
					//do nothing
				}
				else
				{
					count++;
				}
			}
		}
	}

	cout << count << endl;
	time_t end_time = time(0);

	cout << "time : " << end_time - start_time << endl;
	//time took 2
	return 0;

}